aptos-move/block-executor/src/captured_reads.rs

```diff
@@ -327,10 +327,23 @@ impl<T: Transaction> CapturedReads<T> {
     // Return an iterator over the captured reads.
     pub(crate) fn get_read_values_with_delayed_fields(
```

Comment:
The comment might need to be updated to reflect the change in function signature.